src/frontend/templates/footer.html

```diff
@@ -20,7 +20,7 @@
     <div class="footer-top">
         <div class="container footer-social">
             <p class="footer-text">This website is hosted for demo purposes only. It is not an actual shop. This is not a Google product.</p>
-            <p class="footer-text">© 2020 Google Inc (<a href="https://github.com/GoogleCloudPlatform/microservices-demo">Source Code</a>)</p>
+            <p class="footer-text">© 2020-{{ .currentYear }} Google LLC (<a href="https://github.com/GoogleCloudPlatform/microservices-demo">Source Code</a>)</p>
```

Comment:
TBH, I would have removed this "copyright" footer text altogether because it doesn't seem to be a requirement + our CSS/HTML/frontend source code itself contains copyrights + we should remove as much complexity as possible from Online Boutique.

But this works too.

Reply:
Agreed but I didn't want to break the flow too much. This PR does have a net removal of 30 lines of code though, so that's a win :)